### PR TITLE
Atari catch up merge - main 2acf03e850fb883dc6ca5a554f5f827097157ac1

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: chia/full_node/coin_store.py [hunk 1]
```diff
@@ -85,6 +82,13 @@ async def create(cls, db_wrapper: DBWrapper, cache_size: uint32 = uint32(60000))
         self.coin_record_cache = LRUCache(cache_size)
         return self
 
+    async def num_unspent(self) -> int:
+        async with self.coin_record_db.execute("SELECT COUNT(*) FROM coin_record WHERE spent_index=0") as cursor:
```
Comment (Quexington) [inline chia/full_node/coin_store.py:86]: Not an expert in DB performance or if this even matters, but there is a `spent` field on the coin_record too that seems to be the natural tool for the job here.

File: chia/full_node/full_node.py [hunk 2]
```diff
@@ -176,9 +176,7 @@ def sql_trace_callback(req: str):
 
         db_version: int = await lookup_db_version(self.connection)
 
-        self.db_wrapper = DBWrapper(
-            self.connection, self.config.get("allow_database_upgrades", False), db_version=db_version
-        )
+        self.db_wrapper = DBWrapper(self.connection, db_version=db_version)
```
Comment (Quexington) [inline chia/full_node/full_node.py:179]: Is this a correct change?  I thought the "allow_database_upgrades" option was new and as such should probably be in here.

File: chia/util/db_wrapper.py [hunk 3]
```diff
@@ -11,12 +11,10 @@ class DBWrapper:
 
     db: aiosqlite.Connection
     lock: asyncio.Lock
-    allow_upgrades: bool
     db_version: int
 
-    def __init__(self, connection: aiosqlite.Connection, allow_upgrades: bool = False, db_version: int = 1):
+    def __init__(self, connection: aiosqlite.Connection, db_version: int = 1):
```
Comment (Quexington) [inline chia/util/db_wrapper.py:16]: If my above comment was correct, this should also not be a change.

Comment (Quexington) [reply chia/util/db_wrapper.py:16]: There's a couple other instances spread throughout the PR

**Review by Quexington — APPROVED** [review]
A few comments, but none that seem critical to Atari functioning.

File: chia/full_node/coin_store.py (hunk 1 above)
Comment (altendky) [reply chia/full_node/coin_store.py:86]: Introduced in https://github.com/Chia-Network/chia-blockchain/pull/9780 if you want to follow up on the concerns.

File: chia/full_node/full_node.py (hunk 2 above)
Comment (altendky) [reply chia/full_node/full_node.py:179]: Removed in https://github.com/Chia-Network/chia-blockchain/pull/9768.

File: chia/util/db_wrapper.py (hunk 3 above)
Comment (altendky) [reply chia/util/db_wrapper.py:16]: https://github.com/Chia-Network/chia-blockchain/pull/9768

**Pull request merged**